--- a/gcc/config/xtensa/elf.h	2019-12-28 08:47:35.611387257 +0000
+++ b/gcc/config/xtensa/elf.h	2019-12-28 08:50:21.975525455 +0000
@@ -50,11 +50,11 @@
   %{mno-auto-litpools:--no-auto-litpools}"
 
 #undef LIB_SPEC
-#define LIB_SPEC "-lc -lsim -lc -lhandlers-sim -lhal"
+#define LIB_SPEC "-lc"
 
 #undef STARTFILE_SPEC
 #define STARTFILE_SPEC \
-  "crt1-sim%O%s crt0%O%s crti%O%s crtbegin%O%s _vectors%O%s"
+  "crt0%O%s crti%O%s crtbegin%O%s"
 
 #undef ENDFILE_SPEC
 #define ENDFILE_SPEC "crtend%O%s crtn%O%s"  
